src/pages/Governance/hooks/useSubmitStake.ts

```diff
@@ -0,0 +1,32 @@
+import {BCS, TxnBuilderTypes} from "aptos";
+import useSubmitTransaction from "../../../api/hooks/useSubmitTransaction";
+
+const useSubmitStake = () => {
+  const {submitTransaction, transactionResponse, clearTransactionResponse} =
+    useSubmitTransaction();
+
+  async function submitStake(
+    stakingAmount: number,
+    operatorAddr: string,
+    voterAddr: string,
+  ) {
+    const payload = new TxnBuilderTypes.TransactionPayloadScriptFunction(
```

Comment:
I think the function could use some validation before submitting the transaction (e.g. stakingAmount is an integer in a reasonable range, operatorAddr and voterAddr are valid addresses, etc.).

Reply:
added on the `onSubmitClick()`